Packager/CMakeLists.txt

```diff
@@ -18,7 +18,11 @@
 set(PLUGIN_NAME Packager)
 set(MODULE_NAME WPEFramework${PLUGIN_NAME})
 
-list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/cmake")
+if (BUILD_RDK)
```

Comment:
Why do we want to make this explicit ? Why can we not just say:
find_package(libprovision QUIET)

if (libprovision_FOUND)
  ...
else ()
   ...
endif ()

This way it work automagically if you install libprovision